### DISCUSSION TITLE
Disable Coop feature

### DISCUSSION THREAD

**Discussion by SenPr** [BG-Software-LLC/SuperiorSkyblock2#486] [Pending Feature Requests] [answered]
Currently, the only way to disable Coop is to block players from using the commands or setting the co-op limit to 0.

However, this can cause confusion as the role is still displayed in menus such as /island perms
Players would be asking why there is a Coop role but cannot be used and may report it as a mistake/bug.

Would be appreciated if there is something such as disable-coop: true or allowing us to delete the coop role in config.yml, something I tried but the config just regenerate the coop role.

**Comment by Tptuaazn** [comment]
Dont add permissions
superior.island.coop
superior.island.coops

**Reply by SenPr** [reply]
I already mentioned this...
The feature is still there and Coop is can still be seen as a role in commands such as /island perms
Which causes confusion.

**Comment by Lemvs** [comment, marked as answer]
Implemented long time ago.

@OmerBenGera you can mark this answered and close it.
